app/presenters/nsm/tasks/case_outcome.rb

```diff
@@ -7,6 +7,10 @@ class CaseOutcome < Base
       def path
         edit_nsm_steps_case_outcome_path(application)
       end
+
+      def not_applicable?
+        application.before_youth_court_cutoff?
```

Comment:
[relevant to new updates to ticket](https://dsdmoj.atlassian.net/browse/CRM457-2288)